mm2src/coins/solana.rs

```diff
@@ -320,10 +320,9 @@ impl SolanaCoin {
         Ok((hash, fees))
     }
 
-    pub async fn my_balance_spl(&self, infos: &SplTokenInfo) -> Result<CoinBalance, MmError<BalanceError>> {
+    pub async fn my_balance_spl(&self, infos: SplTokenInfo) -> Result<CoinBalance, MmError<BalanceError>> {
         let token_accounts = async_blocking({
             let coin = self.clone();
```

Comment:
optional: I  guess it will also work well without cloning self.

`coin.rpc().get_token_accounts_by_owner(`

Reply:
`coin.rpc().get_token_accounts_by_owner(` is already depend on cloned self.

Reply:
@rozhkovdmitrii can you please give an example on how to do that? FYI when we clone coin here, we clone an Arc, so it's not that big of a problem.

Reply:
I've checked it up again, I was wrong sorry about it )